pkg/shortnames/shortnames_test.go

```diff
@@ -0,0 +1,458 @@
+package shortnames
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/containers/image/v5/docker/reference"
+	"github.com/containers/image/v5/pkg/sysregistriesv2"
+	"github.com/containers/image/v5/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIsShortName(t *testing.T) {
+	tests := []struct {
+		input       string
+		isShortName bool
+		mustFail    bool
+	}{
```

Comment:
It might be good to add to or modify the tests to pull quay.io/skopeo/stable and a variety of it's tags.